terraform/task_definition.tf

```diff
@@ -0,0 +1,339 @@
+    }
+  }
+}
+
+resource "aws_iam_role_policy" "execution_role" {
+  policy = data.aws_iam_policy_document.execution_role.json
+  role   = aws_iam_role.execution_role.id
+}
+
+data "aws_iam_policy_document" "execution_role" {
+  statement {
+    effect    = "Allow"
+    resources = ["*"]
+
+    actions = [
+      "ecr:GetAuthorizationToken",
+      "ecr:BatchCheckLayerAvailability",
+      "ecr:GetDownloadUrlForLayer",
+      "ecr:BatchGetImage",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+      "ssm:GetParameters",
+      "secretsmanager:GetSecretValue",
+    ]
+  }
+}
+
+variable "APP_VERSION" {
+}
+
+variable "WEB_VERSION" {
+}
+
+variable "INFRA_VERSION" {
+}
+
+locals {
+  web = <<EOF
+  {
+    "cpu": 0,
+    "essential": true,
+    "image": "311462405659.dkr.ecr.eu-west-1.amazonaws.com/serve-opg/web:${var.WEB_VERSION}",
+    "mountPoints": [],
+    "name": "web",
+    "portMappings": [{
+      "containerPort": 80,
+      "hostPort": 80,
+      "protocol": "tcp"
+    }],
+    "volumesFrom": [],
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "/ecs/serve-opg",
+        "awslogs-region": "eu-west-1",
+        "awslogs-stream-prefix": "frontend"
+      }
+    },
+    "environment": [{
+      "name": "APP_HOST",
+      "value": "127.0.0.1"
+    },
+    {
+      "name": "APP_PORT",
+      "value": "9000"
+    },
+    {
+      "name": "TIMEOUT",
+      "value": "60"
+    }]
+  }
+
+EOF
+
+
+  app = <<EOF
+  {
+  	"cpu": 0,
+  	"essential": true,
+  	"image": "311462405659.dkr.ecr.eu-west-1.amazonaws.com/serve-opg/app:${var.APP_VERSION}",
+  	"mountPoints": [],
+  	"name": "app",
+  	"portMappings": [{
+      "containerPort": 9000,
+      "hostPort": 9000,
+      "protocol": "tcp"
+  	}],
+  	"volumesFrom": [],
+  	"logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "/ecs/serve-opg",
+        "awslogs-region": "eu-west-1",
+        "awslogs-stream-prefix": "frontend"
+      }
+  	},
+    "secrets": [
+    {
+      "name": "NOTIFICATION_API_KEY",
+      "valueFrom": "${data.aws_secretsmanager_secret.notification_api_key.arn}"
+    },
+    {
+      "name": "OS_PLACES_API_KEY",
+      "valueFrom": "${data.aws_secretsmanager_secret.os_places_api_key.arn}"
+    },
+    {
+      "name": "SIRIUS_PUBLIC_API_PASSWORD",
+      "valueFrom": "${data.aws_secretsmanager_secret.public_api_password.arn}"
+    },
+    {
+      "name": "APP_SECRET",
+      "valueFrom": "${data.aws_secretsmanager_secret.symfony_app_secret.arn}"
+    },
+    {
+      "name": "BEHAT_PASSWORD",
+      "valueFrom": "${aws_secretsmanager_secret.behat_password.arn}"
+    },
+    {
+      "name": "DC_DB_PASS",
+      "valueFrom": "${data.aws_secretsmanager_secret.database_password.arn}"
+    }],
+  	"environment": [{
+  	  "name": "DC_GTM",
+      "value": "${local.dc_gtm}"
+  	},
+  	{
+      "name": "DC_SYMFONY_SECRET",
+      "value": "dc2018!"
```

Comment:
Shouldn't be in clear text, but I can't see `DC_SYMFONY_SECRET` used anywhere. Was it replaced with APP_SECRET?

Reply:
Yeah there's a lot of old terraform that I'll need to take a look at, this included. I've got a ticket in to do a refactor to put things into environment and account folders so will have a review then